File: go/leetcode/easy/26-删除排序数组中的重复项.go
package main

func removeDuplicates(nums []int) int {
	temp := 0
	for i := 0; i < len(nums); i++ {
		if nums[i] != nums[temp] {
			temp++
			nums[temp] = nums[i]
		}
	}
	return temp + 1
}

func main() {

}

File: python/leetcode/algorithms/easy/26-删除排序数组中的重复项.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0
        temp = 0
        for i in range(1, len(nums)):
            if nums[i] != nums[temp]:
                temp += 1
                nums[temp] = nums[i]
        return temp + 1


nums = [0, 0, 1, 1, 1, 2, 2, 3, 3, 4]
s = Solution()
print(s.removeDuplicates(nums))
